package.yaml

```diff
@@ -23,10 +23,11 @@ dependencies:
   - vector <= 1.0
 
 default-extensions:
-  - OverloadedStrings
+  - DerivingStrategies
+  - FlexibleContexts
```

Comment:
ghc 9.2.1 requires this

Reply:
Interesting :thinking:. So before it was somehow implicitly activated, while now this needs to be made explicit?

Reply:
Not sure how it worked before, here's the error I saw:
```sh
[1 of 9] Compiling Language.Souffle.Analysis

lib/Language/Souffle/Analysis.hs:121:10: error:
    • Non type-variable argument
        in the constraint: Category (Analysis m)
      (Use FlexibleContexts to permit this)
    • In the instance declaration for ‘Arrow (Analysis m)’
    |
121 | instance (Monad m, Monoid (m ()), Category (Analysis m)) => Arrow (Analysis m) where
    |          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
Interesting, looks like they fixed a small gap in `FlexibleContexts` then! Probably caused by the `m ()` that's part of the `Monoid` constraint.

Adding the constraint to package.yaml is fine :+1:.